tests/unit/drivers/OdnoklassnikiTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * @link https://github.com/yiimaker/yii2-social-share
+ * @copyright Copyright (c) 2017 Yii Maker
+ * @license BSD 3-Clause License
+ */
+
+namespace ymaker\social\share\tests\unit\drivers;
+
+use ymaker\social\share\base\AbstractDriver;
+use ymaker\social\share\drivers\Odnoklassniki;
+
+/**
+ * Test case for [[Odnoklassniki]] driver.
+ *
+ * @author Petro Sokolnykov <info@xyz.net.ua>
+ * @since 1.0
```

Comment:
Please change @since comment to @since 2.0.
It tag tell to us from what version this code are available.